### PR TITLE
Feature:   Dynamic Contact option for Drives

### PR DESCRIPTION
**Pull request by somnath77**
https://trello.com/c/33k1FBMp

### CODE REVIEW

File: app/controllers/api/v1/admin/users_controller.rb [hunk 1]
```diff
@@ -71,6 +79,12 @@ def user_params
           params.permit(:first_name, :last_name, :email, :role_id,
                         :password, :password_confirmation, :invitation_token, :mobile_number)
         end
+
+        def load_users_with_mobile_no
+          users = User.kept.rolewise_users_list(2)
```
Comment (anilmaurya) [inline app/controllers/api/v1/admin/users_controller.rb:84]: Rename scope to have better visibility of return value

File: app/views/candidate_mailer/invitation_email.html.erb [hunk 2]
```diff
@@ -561,108 +561,42 @@
...
-                                style="
-                                  border-bottom: 2px solid
-                                    rgba(57, 250, 255, 0.22);
-                                  padding-top: 8px;
-                                  padding-bottom: 8px;
-                                  padding-right: 16px;
-                                "
-                              >
-                                <p
+                              <% @drive.users.limit(3).each do |user| %>
```
Comment (anilmaurya) [inline app/views/candidate_mailer/invitation_email.html.erb:564]: Instead of `drive.users`, we should call it `drive.admin_users`

Comment (anilmaurya) [reply app/views/candidate_mailer/invitation_email.html.erb:564]: Remove logic for picking top 3 from template

File: app/controllers/api/v1/admin/users_controller.rb (hunk 1 above)
Comment (somnath77) [reply app/controllers/api/v1/admin/users_controller.rb:84]: done

File: app/views/candidate_mailer/invitation_email.html.erb (hunk 2 above)
Comment (somnath77) [reply app/views/candidate_mailer/invitation_email.html.erb:564]: done

**Review by onkar-josh — APPROVED** [review]

**Pull request merged**